README.md

```diff
@@ -13,6 +13,7 @@ This client supports the following Google Cloud Platform services:
 * [Google BigQuery](#google-bigquery)
 * [Google Cloud Datastore](#google-cloud-datastore)
 * [Google Cloud DNS](#google-cloud-dns)
+* [Google Cloud Logging](#google-cloud-logging)
```

Comment:
Should we denote that this is API is still in beta? I believe for pubsub we used to add the text *(beta)* after the link.